content/blog/2016-07-23-pull-does-not-scale-or-does-it.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Pull doesn't scale - or does it?
+created_at: 2016-07-23
+kind: article
+author_name: Julius Volz
+---
+
+Let's talk about one particularly hard-to-kill myth again. Whenever there is a
```

Comment:
Hmm, I think I'm fine with using the "common person's" definition of "myth", but let's see what others say.